bluepysnap/simulation.py

```diff
@@ -126,7 +126,7 @@ def simulator(self):
     def node_sets(self):
         """Returns the NodeSets object bound to the simulation."""
         node_sets_file = self.to_libsonata.node_sets_file
-        return NodeSets(node_sets_file) if node_sets_file else {}
+        return NodeSets.from_file(node_sets_file) if node_sets_file else {}
```

Comment:
This wasn't changed in this PR, but it would be more consistent if we always return a NodeSet instance, instead of returning an empty dict when `node_sets_file` is empty.
What about of something like this?
```python
        return NodeSets.from_file(node_sets_file) if node_sets_file else NodeSets.from_string("{}")
```
or 
```python
        return NodeSets.from_file(node_sets_file) if node_sets_file else NodeSets.from_dict({})
```

Reply:
Alternatively, we could return None, but it seems a bigger breaking change.

Reply:
Good, this is fixed in `Circuit.node_sets`, but it should be changed in `Simulation.node_sets` as well (and the changelog updated to mention Simulation)

Reply:
🤦‍♂️ , I remembered that when there is no simulation node set file defined, libsonata returns circuits node set path so this needed no fixing. OFC, both can be empty. 

Thanks > should be fixed now.